.ahoy.yml

```diff
@@ -121,8 +119,40 @@ commands:
   test-bdd:
     usage: Run BDD tests.
     cmd: |
+      ahoy start-ckan-job-workers &
```

Comment:
This probably doesn't need to be backgrounded when the workers are individually backgrounded.

Reply:
@ThrawnCA Yes you are correct and I have removed it.